src/webgpu/api/operation/render_pass/clear_value.spec.ts

```diff
@@ -119,7 +124,10 @@ g.test('stencil_clear_value')
 
     const stencilAspectSizeInBytes = depthStencilFormatAspectSize(stencilFormat, 'stencil-only');
     assert(stencilAspectSizeInBytes > 0);
-    const stencilReference = stencilClearValue & ((stencilAspectSizeInBytes << 8) - 1);
+    const expectedStencilValue = stencilClearValue & ((stencilAspectSizeInBytes << 8) - 1);
+    const stencilReference = applyStencilClearValueAsStencilReferenceValue
```

Comment:
Maybe it is worth to add a comment to point out that `stencilReference` used in `setStencilReference` will also be masked to the lowest valid bits, so no matter what we set in the rest high bits that will be masked out (different or same between `stencilClearValue` and `stencilReference`), the test will pass iff the valid lowest bits are the same.

Reply:
Done